R/plot.check_normality.R

```diff
@@ -31,6 +34,7 @@ plot.see_check_normality <- function(x,
                                      dot_alpha = 0.8,
                                      colors = c("#3aaf85", "#1b6ca8"),
                                      detrend = FALSE,
+                                     method = "ell",
```

Comment:
Should we add `match.arg` for valid options, as well?

Reply:
I don't think so - that already happens in `qqplotr`.